services/bettersocial/user/BetterSocialBlockUserV2.js

```diff
@@ -13,6 +13,91 @@ const Getstream = require('../../../vendor/getstream');
 const RedisBlockHelper = require('../../redis/helper/RedisBlockHelper');
 const BetterSocialScoreBlockUser = require('../score/block-user');
 
+const getstream_unfollow_sequence = async (token, feedOwner, feedTarget, direct = false) => {
+  try {
+    // Main feed following unfollow user anon feed target
+    if (direct) {
+      await Getstream.feed.unfollowMainFeedFollowingDirect(feedOwner, feedTarget);
+    } else {
+      await Getstream.feed.unfollowMainFeedFollowing(token, feedOwner, feedTarget);
+    }
+    // Main feed unfollow user feed target
+    await Getstream.feed.unfollowUser(feedOwner, feedTarget);
+    // Main feed unfollow user exclusive feed target
+    await Getstream.feed.unfollowUserExclusive(feedOwner, feedTarget);
+  } catch (e) {
+    console.log('Error in block user v2 getstream');
+    console.log(e);
+    return {
+      isSuccess: false,
+      message: e?.message || 'Error in block user v2 getstream'
+    };
+  }
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getstream_unfollow_sequence` does not return a success status when the operations are successful. This could lead to confusion when using this function as it only returns a failure status. Consider adding a return statement indicating success after all operations have been completed.
```diff
+    return {
+      isSuccess: true,
+      message: 'Unfollow sequence completed successfully'
+    };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->